packages/react-components/react-tabster/src/focus/createFocusOutlineStyle.ts

```diff
@@ -34,7 +34,7 @@ export interface CreateFocusOutlineStyleOptions extends Omit<CreateCustomFocusIn
  * @param options - Configures the style of the focus outline
  * @returns focus outline styles object
  */
-const getFocusOutlineStyles = (options: FocusOutlineStyleOptions): GriffelStyle => {
+export const getFocusOutlineStyles = (options: FocusOutlineStyleOptions): GriffelStyle => {
```

Comment:
Are you sure this is needed? In the past year we've gotten to the point where all browsers now support the native outline that follows the border radius,

Native outline should be a better choice because there's no need to have a positioned HTML element for the  pseudo element based styles to work at all.

Also native outline will have better perf, since the styles will not cause reflow

Reply:
I'm just using whatever is being used for current outline for the focused element, or am I missing something? :)

Reply:
I totally understand your point, I just wasn't aware we are shifting towards native outline and this is deprecated.

I'll close this PR like nothing ever happened 😄